jupyterhub/templates/hub/deployment.yaml

```diff
@@ -105,6 +105,13 @@ spec:
         - name: {{ $key | quote }}
           value: {{ $value | quote }}
         {{ end -}}
+        {{ if .Values.auth.state.enable }}
```

Comment:
This should probably be put in the hub secret and then put here via a ref I think.

Reply:
I think I already do that. Can you clarify?

Reply:
Nope, I just hadn't fully woken up yet. Aaa, sorryy!!